src/modes/hip/utils.cpp

```diff
@@ -54,8 +54,8 @@ namespace occa {
 
     void enablePeerToPeer(hipCtx_t context) {
 
-      OCCA_HIP_ERROR("Enabling Peer-to-Peer",
-                     hipCtxEnablePeerAccess(context, 0) );
+      // OCCA_HIP_ERROR("Enabling Peer-to-Peer",
```

Comment:
Should we throw an exception here?

Reply:
The hipCtx_t struct (analogue of a CUcontext) is actually marked as deprecated when compiling HIP for AMD targets. The reason is because its not really necessary for AMD's APIs. 

It is inconvenient to have this deprecate when compiling for Nvidia targets, though. But I assume it means the `hipCtxEnablePeerAccess` doesn't actually accomplish anything with AMD GPUs. AMD GPUs can access other GPU's memory when able (using flat addressing, I believe) by default. I can check on this though. 

In the meantime, I'm not sure if this should throw. Would it be better for switching between cuda<->hip if it just always returned successfully?